### ISSUE TITLE
Mac OS customer theme don't work

### ISSUE THREAD

**Issue by cmi1993** [samuele-cozzi/obsidian-marp-slides#76]
how to config the theme path in obsidian Marp plugins?
I use both relative and absolute path ,but do not work either?

**Comment by ishy-k** [comment]
Same here. It is reproducible for me on MacOS. I tried the following:

- Relative path to vault.
- Absolute path.
- Windows syntax for relative and absolute path.
- Different names for `theme.css`.

Nothing seems to work. Can I provide any help regarding fixing this?

**Comment by samuele-cozzi** [comment]
Thank you very much for your interest in the project. I've finally managed to run Mac OS Catalina on Ubuntu using QEMU.

My custom themes work with the following configuration. I have created a folder in the vault where I will put all my custom themes:

![Screenshot from 2023-12-16 17-19-01](https://github.com/samuele-cozzi/obsidian-marp-slides/assets/2479040/78e9e75d-286e-4d69-b0af-c37e4e8853fd)

Then I have write in theme path configuration "templates/themes" folder:

![Screenshot from 2023-12-16 17-20-25](https://github.com/samuele-cozzi/obsidian-marp-slides/assets/2479040/298d464b-f005-4f8d-b471-8b8e76005405)

Let me know if it works also in a original MacOS
